wallet/src/main/scala/org/bitcoins/wallet/models/AddressTable.scala

```diff
@@ -118,29 +136,27 @@ object AddressDbHelper {
 class AddressTable(tag: Tag) extends Table[AddressDb](tag, "addresses") {
   import org.bitcoins.db.DbCommonsColumnMappers._
 
-  def purpose: Rep[HDPurpose] = column[HDPurpose]("hd_purpose")
+  def purpose: Rep[HDPurpose] = column("hd_purpose")
+
+  def accountIndex: Rep[Int] = column("account_index")
 
-  def accountIndex: Rep[Int] = column[Int]("account_index")
+  def accountCoin: Rep[HDCoinType] = column("hd_coin")
 
-  def accountCoin: Rep[HDCoinType] = column[HDCoinType]("hd_coin")
+  def accountChainType: Rep[HDChainType] = column("hd_chain_type")
 
-  def accountChainType: Rep[HDChainType] =
-    column[HDChainType]("hd_chain_type")
+  def addressIndex: Rep[Int] = column("address_index")
 
-  def addressIndex: Rep[Int] = column[Int]("address_index")
+  def address: Rep[BitcoinAddress] = column("address", O.PrimaryKey)
 
-  def address: Rep[BitcoinAddress] =
-    column[BitcoinAddress]("address", O.PrimaryKey)
+  def ecPublicKey: Rep[ECPublicKey] = column("pubkey")
 
-  def ecPublicKey: Rep[ECPublicKey] = column[ECPublicKey]("pubkey")
+  def hashedPubKey: Rep[Sha256Hash160Digest] = column("hashed_pubkey")
```

Comment:
nit: but this probably should be changed to `hash` and be of type `Rep[HashDigest]`